src/content/mainpagecards/annotated-research-context.md

```diff
@@ -11,4 +11,4 @@ text-position: left
 
 ---
 
-DataPLANT will allow users to store their data in form of an annotated research context. This includes not only the measurement data, but also (meta)data annotations, tools, and scripts. This will allow reproducibility plus to forward “raw data” and metadata slices to public repositories (e.g. EBI NCBI etc.).
\ No newline at end of file
+DataPLANT will enable users to store their data in form of an Annotated Research Context (ARC). This includes not only the measurement data, but also (meta)data annotations, tools, and scripts. This will allow reproducibility and also forwarding “raw data” and metadata slices to public repositories (e.g. EBI, NCBI etc.).
```

Comment:
"DataPLANT enables users to store..."

_will enable_ sounds like we we plan to do that in the future, but since it's already happening I suggest to use present tense